samples/graph-proactive-installation/nodejs/bots/proactiveBot.js

```diff
@@ -0,0 +1,74 @@
+const { ActivityHandler, TurnContext, TeamsInfo, MessageFactory } = require('botbuilder');
+var ProactiveAppIntallationHelper = require('../Models/ProactiveAppIntallationHelper');
+
+class ProactiveBot extends ActivityHandler {
+    constructor(conversationReferences) {
+        super();
+        this.conversationReferences = conversationReferences;
+        this.onConversationUpdate(async (context, next) => {
+            this.addConversationReference(context.activity);
+        });
+
+        this.onMembersAdded(async (context, next) => {
+            const membersAdded = context.activity.membersAdded;
+            for (let cnt = 0; cnt < membersAdded.length; cnt++) {
+                if (membersAdded[cnt].id !== context.activity.recipient.id) {
+                    this.addConversationReference(context.activity);
+                }
+            }
+            await next();
+        });
+
+        this.onMessage(async (context, next) => {
+            TurnContext.removeRecipientMention(context.activity);
+            const text = context.activity.text.trim().toLocaleLowerCase();
+            if (text.includes('install')) {
+                await this.InstallAppInTeamsAndChatMembersPersonalScope(context);
+            }
+            else if (text.includes('send')) {
+                await this.SendNotificationToAllUsersAsync(context);
+            }
+        });
+    }
+
+    async InstallAppInTeamsAndChatMembersPersonalScope(context) {
+        let NewAppInstallCount = 0;
+        let ExistingAppInstallCount = 0;
+        let result = "";
+        const TeamMembers = await TeamsInfo.getPagedMembers(context);
+        let Count = TeamMembers.members.map(async member => {
+            if (!this.conversationReferences[member.aadObjectId]) {
+                result = await ProactiveAppIntallationHelper.InstallAppInPersonalScope(context.activity.conversation.tenantId, member.aadObjectId);
+            }
+            return result;
+        });
+
+        (await Promise.all(Count)).forEach(function (Status_Code) {
+            if (Status_Code == 409) ExistingAppInstallCount++;
```

Comment:
Fix indentation

Reply:
Fixed the Formatting in Proactivebot and Proactiveinstalltionhelper files